src/leiningen/ring/server_headless.clj

```diff
@@ -6,4 +6,4 @@
   ([project]
      (server-task project {:open-browser? false}))
   ([project port]
-     (server-task project {:port (Integer. port), :open-browser? false})))
+     (server-task project {:port (int port), :open-browser? false})))
```

Comment:
What's the purpose of this change?

Reply:
this is wrong - i've already reverted this in server.clj - will revert here too (just needs a type-hint)